rootful_network.py

```diff
@@ -565,22 +565,27 @@ def setup(args, con):
         "container_id": container.id,
         "container_name": container.name,
         "port_mappings": portmappings(args.publish),
-        "networks": {
-            network.name: {
+        "networks": {},
+        "network_info": {},
+    }
+
+    interface_increment = 0
+    for podman_network in args.networks:
+        with podman.PodmanClient(base_url="unix:///run/podman/podman.sock") as root_podman:
```

Comment:
I would have switched those to avoid opening the connection to the podman socket multiple times, but since nobody will be passing 12,000 networks to this, the difference probably doesn't matter.